test/functional/connection_tests.js

```diff
@@ -231,6 +231,7 @@ describe('Connection', function() {
    */
   function connectionTester(configuration, testName, callback) {
     return function(err, client) {
+      expect(err).to.not.exist;
```

Comment:
Is this change necessary?

Reply:
I'm currently getting a `configuration.url() is not a function` error for many of the tests, after it gets about halfway through. This fix was to catch the errors - once it's fixed I'll remove it!